presto-hive/src/test/java/com/facebook/presto/hive/TestHiveMaterializedQueries.java

```diff
@@ -88,6 +92,102 @@ public void testValues()
         // decimal type is not supported by the Hive hash code function
     }
 
+    @Override
+    public void testAntiJoinNullHandling()
+    {
+        // Unsupported Hive type: unknown
+    }
+
+    @Override
+    public void testApproxSetBigintGroupBy()
+    {
+        // Unsupported Hive type: HyperLogLog
+    }
+
+    @Override
+    public void testApproxSetVarcharGroupBy()
+    {
+        // Unsupported Hive type: HyperLogLog
+    }
+
+    @Override
+    public void testApproxSetDoubleGroupBy()
+    {
+        // Unsupported Hive type: HyperLogLog
+    }
+
+    @Override
+    public void testApproxSetGroupByWithOnlyNullsInOneGroup()
+    {
+        // Unsupported Hive type: HyperLogLog
+    }
+
+    @Override
+    public void testApproxSetGroupByWithNulls()
+    {
+        // Unsupported Hive type: HyperLogLog
+    }
+
+    @Override
+    public void testMergeHyperLogLogGroupBy()
```

Comment:
Just checking my understanding, is it because we are materializing to ORC format and there is no type in Hive for HLL?  If so I suppose with "raw" format we could enable them?

Reply:
Unfortunately Hive cannot serialize HLL type into a table. That's why these queries are not yet supported.